cleanco/clean.py

```diff
@@ -65,17 +69,17 @@ def normalized(text):
     return remove_accents(text)
 
 
-def prepare_default_terms():
+def prepare_default_terms(country: Optional[str] = None):
```

Comment:
Should we then do typing for all the functions we change? Because changing everything seems like a lot of work, but not having a return type kind of defeats the point.